src/include/kdbprivate.h

```diff
@@ -235,6 +262,89 @@ struct _Key
 	 */
 	size_t keyUSize;
 
+	/**
+	 * Reference counter
+	 */
+	uint16_t refs;
+
+	uint16_t flags;
+};
+
+// private methods for COW keys
```

Comment:
Can you put all the stuff here in a separate header file? (See doc/decisions/3_decided/header_include.md)

Reply:
> See doc/decisions/3_decided/header_include.md

That's not even close to being implemented. I don't see any benefit to moving things. We can't even move them to the correct location, because the folder structure is still all wrong...

Reply:
Okay, so we keep it in `kdbprivate.h` for the time being?